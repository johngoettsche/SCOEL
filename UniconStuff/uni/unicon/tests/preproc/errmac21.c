/* test error message: identifier missing on #undef */

#undef
x
