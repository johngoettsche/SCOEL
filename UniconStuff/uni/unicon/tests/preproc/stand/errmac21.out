pp: File errmac21.c; Line 3: #undef requires an identifier argument
